app/src/main/java/layout/mapFragment.java

```diff
@@ -65,13 +66,18 @@
      * Flag indicating whether a requested permission has been denied after returning in
      * {@link #onRequestPermissionsResult(int, String[], int[])}.
      */
-    private boolean mPermissionDenied = false;
+    private boolean permissionDenied = false;
+    private static String[] PERMISSIONS_LOCATION = {
+            Manifest.permission.ACCESS_FINE_LOCATION
+    };
 
     private GoogleMap mMap;
 
     private GoogleApiClient mGoogleApiClient;
     private Location mLastKnownLocation;
 
+    private static final String frag = "map";
+
```

Comment:
For the purposes of updating the "frag" field in FeelTripApplication, which then allows us to know which frag we're currently viewing from any point in the app.